pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 
 name = "fmmax"
-version = "0.4.0"
+version = "v0.4.0"
```

Comment:
So we just to be sure to bump the version here as needed, right? There aren't any other places to be aware of?